external-crates/move/crates/move-analyzer/src/completion.rs

```diff
@@ -239,7 +239,7 @@ fn call_completion_item(
     method_name: &Symbol,
     function_name: &Symbol,
     type_args: &[Type],
-    arg_names: &[Symbol],
```

Comment:
What's the motivation for changing these from `Symbol` to `Name`? is the location information on the names relevant?

Reply:
Ah, I see they are -- we use them as def locs in the inlay hints.